ui/src/shared/components/PostComponents.tsx

```diff
@@ -97,12 +100,17 @@ const PostLink = styled.a.attrs({ target: "_blank", rel: "noreferrer noopener" }
 `;
 
 export const PostLinks: React.FC<PostLinksProps> = ({ data }) => {
-  const { job_link, company_link } = data;
-  if (!job_link && !company_link) return null;
-  return (
-    <PostLinksContainer>
-      {job_link && <PostLink href={job_link}>Job Details</PostLink>}
-      {company_link && <PostLink href={company_link}>Company Info</PostLink>}
-    </PostLinksContainer>
-  );
+  console.log(data);
+  /* eslint-disable @typescript-eslint/camelcase */
+  const { job_link, company_link, website } = data;
+  if (job_link || company_link || website) {
+    return (
+      <PostLinksContainer>
+        {job_link && <PostLink href={job_link}>Job Details</PostLink>}
+        {company_link && <PostLink href={company_link}>Company Info</PostLink>}
+        {website && <PostLink href={website}>My Portfolio</PostLink>}
```

Comment:
Similar to this bug @ng95junwei mentioned, if we have a URL without the http/https protocol (e.g. www.getajob.com), the URL becomes a relative link.

I think we can either get users to fill in a correct URL (via validation on Front End) or accept links and convert it ourselves either on Back End or when displaying it (conversion to a proper URL)

@subhransu what do you think? 

<img width="1266" alt="Screenshot 2020-10-24 at 12 39 31 AM" src="https://user-images.githubusercontent.com/6391234/97031420-d5203b80-1592-11eb-8a0f-4c8eb854aa02.png">
^ See bottom left

Reply:
Let's carry on this discussion separately, not part of this PR's scope.